VK_V1.0/VK_V1.0/Model/News/Comment.swift

```diff
@@ -0,0 +1,10 @@
+// Comment.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Комментарии
+struct Comment: Decodable {
```

Comment:
```suggestion
struct Comment: Decodable {
```
Я бы обозвала с s, не знаю, насколько это корректно, но по идее в jsonе этот элемент отображает количество комментариев и можно ли их оставлять, тоесть она относится не к одному комментарию, а ко всем